package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "foreman_webhooks",
+  "version": "1.0.0",
```

can you match the version in https://github.com/theforeman/foreman_webhooks/blob/master/lib/foreman_webhooks/version.rb#L4

```suggestion
  "version": "0.0.1",
```